api-inference-community/api_inference_community/normalizers.py

```diff
@@ -0,0 +1,41 @@
+"""
+Helper classes to modify pipeline outputs from tensors to expected pipeline output
+"""
+
+from typing import TYPE_CHECKING, Dict, List, Union
+
+
+Classes = Dict[str, Union[str, float]]
+
+if TYPE_CHECKING:
+    try:
+        import torch
+    except Exception:
+        pass
+
+
+def speaker_diarization_normalize(
+    tensor: "torch.Tensor", sampling_rate: int, classe_names: List[str]
```

Comment:
nit: "classes_names" and same below

Reply:
I really would like to avoid double ss in names, it's very confusing. 
-1 "s" at the end of a variables -> helps understand it's a list
- otherwise plurals are to be avoided IMO as it's very inconsistent for non natives (even for natives sometimes)

`class_names` or `classnames` if you prefer.

Reply:
I opted for `classnames` right now (classe is french so it was wrong anyway)